pkg/skaffold/build/local/jib.go

```diff
@@ -18,16 +18,110 @@ package local
 
 import (
 	"context"
+	"crypto/sha1"
+	"encoding/hex"
 	"io"
+	"os/exec"
+	"regexp"
 
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/constants"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/v1alpha3"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 )
 
-func (b *Builder) buildJibMaven(_ /*ctx*/ context.Context, _ /*out*/ io.Writer, _ /*workspace*/ string, _ /*a*/ *v1alpha3.JibMavenArtifact) (string, error) {
-	return "", errors.New("buildJibMaven is unimplemented")
+func (b *Builder) buildJibMaven(ctx context.Context, out io.Writer, workspace string, a *v1alpha3.JibMavenArtifact) (string, error) {
```

Comment:
we should pull out maven related stuff to `jib_maven.go` and gradle methods to `jib_gradle.go`?